Designer/Block.csv

```diff
@@ -1,10 +1,10 @@
 ID,Name,AddsToBiome,GatherTool,GatherEffect,ItemID,CollectibleID,IsFlammable,IsLiquid,MaxToughness

```

Comment:
nit: I feel as though the Name field should be capitalized, in the same way the other csvs capitalize name. That is, "grey brick" would become "Grey brick", and it would be up to a context sensitive parser to know how to capitalize it.

Also, you may want to introduce a column for the plural of an object. That's neither here nor there, just thinking of it for now.

Reply:
1. Notes!
2. Good call, yeah I should consider that.